run/localizer/locales.ts

```diff
@@ -0,0 +1,931 @@
+// This file was generated by a script. Do not modify this file manually.
```

Comment:
Does this need to be updated via the script? Last updated 1 month ago

Reply:
I don't actually know, will make a note to check this

Reply:
Have created a ticket so I don't forget https://optf.atlassian.net/browse/QA-353